relay-general/src/protocol/device_class.rs

```diff
@@ -50,3 +47,57 @@ impl fmt::Display for DeviceClass {
         self.0.to_string().fmt(f)
     }
 }
+
+static APPLE_DEVICE_MODEL_TO_CLASS_MAP: Lazy<BTreeMap<&str, DeviceClass>> = Lazy::new(|| {
+    let mut map = BTreeMap::new();
+    map.insert("iPhone1,1", DeviceClass::LOW);
+    map.insert("iPhone1,2", DeviceClass::LOW);
```

Comment:
You could simplify this and make it more readable with `BTreeMap::from`, see example in [rust playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=cfdd8ef7be54e9d20b3ba71863174b85).